EasyReflectometryApp/Gui/Pages/Analysis/SideBarAdvanced.qml

```diff
@@ -157,9 +157,25 @@ EaComponents.SideBarColumn {
             topPadding: 0
             id: rq4
             checked: ExGlobals.Constants.proxy.simulation.plotRQ4
-            text: qsTr("Show R(q)q⁴")
+            text: qsTr("Show as R(q)q⁴")
             onToggled: ExGlobals.Constants.proxy.simulation.setPlotRQ4()
         }
+
+        EaElements.CheckBox {
+            topPadding: 0
+            id: scale
```

Comment:
It seems that you are not referring to this element, so there is no need to specify its `id` explicitly. The same is true for `rq4` and `bkg`.